go/keybase/main.go

```diff
@@ -46,8 +46,19 @@ func keybaseExit(exitCode int) {
 	os.Exit(exitCode)
 }
 
+// Preserve non-critical errors that happen very early during startup,
+// where logging is not set up yet, to be printed later when logging
+// is functioning.
+var startupErrors []error
+
 func main() {
-	err := libkb.SaferDLLLoading()
+	if err := libkb.SaferDLLLoading(); err != nil {
+		// Don't abort here. This should not happen on any known
+		// version of Windows, but new MS platforms may create
+		// regressions.
+		startupErrors = append(startupErrors,
+			fmt.Errorf("SaferDLLLoading error: %v", err.Error()))
```

Comment:
cc @zanderz previously it called `g.Log.Debug` here which would go nowhere because logging was not initialised yet. LMK if this looks good.